core-bundle/contao/languages/en/default.xlf

```diff
@@ -1074,7 +1074,7 @@
         <source>Fit the box</source>
       </trans-unit>
       <trans-unit id="MSC.box.1">
-        <source>The shorter side of the image is adjusted to the given dimensions and the image is resized proportionally.</source>
+        <source>The image will be resized proportionally so that both sides do not exceed the given dimensions.</source>
```

Comment:
```suggestion
        <source>The image will be resized proportionally so that both sides do not exceed the given dimensions. Same as &amp;gt;code&amp;lt;contain&amp;gt;/code&amp;lt; in CSS.</source>
```

Reply:
I used `The image will be resized proportionally so that both sides do not exceed the given dimensions. Same as &lt;em&gt;contain&lt;/em&gt; in CSS.` in that example before reading this :)

Reply:
Should we still change it to code or keep it at `em`?

Reply:
I’m not sure, we have to use the same as other explanations do I think. AFAIR the helptext for `sizes` also has such code references.

Reply:
I think you are right, `<em>` seems to be the correct one.

Reply:
🙈